assets/models/system/Mixtral-8x7B-v0-1/spec.yaml

```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mistralai-Mixtral-8x7B-v01
+path: ./
+properties:
+  inference-recommended-sku: Standard_NC24s_v3, Standard_NC24rs_v3,
+    Standard_ND40rs_v2, Standard_NC24ads_A100_v4, Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96amsr_A100_v4, Standard_ND96asr_v4    
+  languages: EN
+tags:
+  Featured: ''
+  SharedComputeCapacityEnabled: ''
+  inference_compute_allow_list:
+  - Standard_NC24s_v3
+  - Standard_NC24rs_v3
+  - Standard_ND40rs_v2
+  - Standard_NC24ads_A100_v4
+  - Standard_NC48ads_A100_v4
+  - Standard_NC96ads_A100_v4
+  - Standard_ND96amsr_A100_v4
+  - Standard_ND96asr_v4
+  model_specific_defaults:
+    precision: '16'
+  inference_supported_envs:
+  - vllm
+  license: apache-2.0
```

Comment:
LICENSE is missing in model file

Reply:
added. BTW I don't see in some of the other models either, please check those.. (e.g. Mistral-7B)

Reply:
Thanks @msp8955. Yes we have found few models are missing LICENSE file. Hence making sure we dont make more such errors. Validation gate to catch in future